### ISSUE TITLE
Is Philips Go really compatible with Deconz?

### ISSUE THREAD

**Issue by SnaKKo** [dresden-elektronik/deconz-rest-plugin#2718]
Hi,
Does Philips Go works correctly with Deconz?
I'm reading a lot of issues here on Github about it.

Anyone tried succesfully? Do you have problems with Philips Go/Deconz? Thanks

**Comment by zladukas** [comment]
I have first version of Philips HUE Go and have no problems (like any other Hue product) at all.

**Comment by SnaKKo** [comment]
Thansk @zladukas , are you able to manage the Hue Go (with Deconz) running on battery?
Have you "paired" the Hue Go and Deconz without installing the Hue app?

**Comment by zladukas** [comment]
Yes. Paired, installed newer FW once, when able to download. You don't need Hue app. But I have no expirience with newer one, whitch has bluetooth or smth...

**Comment by Kane610** [comment]
On battery I don't think the device is listening to Zigbee any more, but I haven't verified that

**Comment by Gizmo-Ger** [comment]
I have one of the newer versions and I can´t get it to work with Deconz. With or without on battery.

**Comment by ebaauw** [comment]
Does it work with the Hue bridge?

**Comment by Gizmo-Ger** [comment]
> Does it work with the Hue bridge?

Since I don´t have a Hue bridge I´m using Hue Essentials on the phone. Pairing was no problem at all. I´ve tried several times to pair it with deconz without any luck.

**Comment by ebaauw** [comment]
That connects to the Go over Bluetooth, doesn’t it?  Isn’t their a setting to prep the Go for pairing with the bridge?

**Comment by Gizmo-Ger** [comment]
Yes, it´s connected via Bluetooth in the Philips Hue BT app on IOS (Model LCT026, SW 1.69.9_hB3217DF4) but trying to add Hue Bridge in the app it doesn´t find the Deconz Bridge. I´ve tried unlock Gateway and/or open network. Both approaches don´t work.

**Comment by SnaKKo** [comment]
So it seems that just the old one works correctly, the newer one have some issues with Deconz.

**Comment by Gizmo-Ger** [comment]
maybe it´s the same like https://github.com/Koenkk/zigbee-herdsman-converters/pull/801#issuecomment-566182010 ?

**Comment by Kane610** [comment]
You probably need to reset it first. It can't be paired over Bluetooth and zigbee at the same time

**Comment by Gizmo-Ger** [comment]
I did reset the Go and made sure no Bluetooth connection was established. Still no luck with adding the Go to Deconz.

**Comment by wvuyk** [comment]
Trying to understand this. Where do you want to add the Deconz gateway to? 
Is it the Philips Hue app? As far I know you cannot add the current Deconz gateway to the Philips Hue app anymore.  Certainly not the BT version?

**Comment by Gizmo-Ger** [comment]
No, I am not using the Philips Hue app at all. Most of my light bulbs are Ikea Tradfri and I swapped the Ikea gateay for the Conbee II. I control the lights via Alexa or Phoscon, except for the Philips Go which seems to be stubborn.

**Comment by wvuyk** [comment]
> but trying to add Hue Bridge in the app it doesn´t find the Deconz Bridge

But what is the app you are referring here to then? Is that Phoscon? Does it not find its own bridge?

**Comment by Gizmo-Ger** [comment]
Sorry for the mix up. All I want to do is using the Philips Hue Go with Phoscon. So far I could use the Go with the Hue BT app and Hue Essentials. Switching from bluetooth connection to Zigbee doesn´t work. In order to switch to Zigbee I need to add a bridge in the app but the app doesn´t find the Phoscon gateway. Like I said I don´t have a hue bridge, just the ConbeeII stick.
